server/routes/locations.js

```diff
@@ -0,0 +1,44 @@
+const { isNilOrEmpty } = require('../utils/utils')
+const { processInput } = require('../services/validation')
+const { full } = require('../config/incident')
+
+module.exports = ({ locationService }) => {
+  // get list of all prisons
+  const viewPrisons = editMode => async (req, res) => {
+    const { token } = res.locals.user
+    const prisons = await locationService.getActiveAgenciesByType(token, 'INST')
```

Comment:
Might be safer to request tokens as a system to system request rather than as a specific user.
if you look to see how `systemToken` is used, where we pass the user's username.

Reply:
ok